master/docs/manual/cfg-wwwhooks.rst

```diff
@@ -77,7 +79,9 @@ The parameters are:
 
 :guilabel:`Secret`
     Any value.
-    If you provide a non-empty value (recommended), make sure that your hook is configured to use it::
+    If you provide a non-empty value (recommended), make sure that your hook is configured to use it:
+
+.. code-block:: python
```

Comment:
the `.. code-block:: python` along with the actual code should be indented on the same leve as "If you provide", your change brings the code example one indentation level less.